src/java/picard/vcf/SplitVcfs.java

```diff
@@ -30,9 +30,21 @@
  * extension will create gzip-compressed output.
  */
 @CommandLineProgramProperties(
-        usage = "Splits an input VCF or BCF file into two VCF files, one for indel records and one for SNPs. The" +
+        usage = "Splits an input VCF or BCF file into two VCF files, one for INDEL records and one for SNPs. The" +
                 "headers of the two output files will be identical. An index file is created and a" +
-                "sequence dictionary is required by default.",
+                "sequence dictionary is required by default. To avoid exceptions, if events other than SNPs or" +
+                " INDELs are present, set the STRICT option to \"false\".*" +
```

Comment:
Does this mean this is necessary to not choke on mixed records? So what happens to mixed records? In GATK we treat them as indels...